apps/mobile/src/libs/enums/enums.ts

```diff
@@ -1,9 +1,16 @@
 export {
+	AngleGradient,
 	BaseColor,
 	DataStatus,
 	GradientColor,
 	ToastMessageTitle,
 	ToastMessageType,
 } from "./app/app";
 export { RootScreenName } from "./navigation/navigation";
-export { APIPath, AppEnvironment, ContentType, ServerErrorType } from "shared";
+export {
+	APIPath,
+	AppEnvironment,
+	ConfirmPasswordCustomValidation,
```

Comment:
It is a part of auth module, so export from auth module on mobile as well

Reply:
Not fixed

Reply:
I moved AgleGradient to the `~/libs/enums/auth` and exported from `./auth/auth` , did I understand you right?

Reply:
No, not AngleGradient. AngleGradient is not auth specific, it should be global. ConfirmPasswordCustomValidation is auth specific

Reply:
Thank you, I misunderstood.